tests/utils/command_line_params.cpp

```diff
@@ -145,7 +145,7 @@ QudaTransferType staggered_transfer_type = QUDA_TRANSFER_COARSE_KD;
 // we only actually support 4 here currently
 quda::mgarray<std::array<int, 4>> geo_block_size = {};
 
-#if (CUDA_VERSION >= 10010 && __COMPUTE_CAPABILITY__ >= 700)
+#ifdef QUDA_MMA_AVAILABLE
```

Comment:
Just a thought: we are removing `CUDA_VERSION >= 10010` as we now require CUDA 11.0, should we also remove it in the definition of `QUDA_MMA_AVAILABLE`?

Reply:
IIRC we still technically support Pascal? I've lost track, tbh.

Reply:
Yes, but can and should remove the CUDA_VERSION here.

Reply:
7c0a6bd

Reply:
What happens on sm60 here?